app/drivers/tools/repair/java/Recoder.py

```diff
@@ -27,23 +27,44 @@ def run_repair(self, bug_info, repair_config_info):
 
         timeout_h = str(repair_config_info[self.key_timeout])
 
-        if not self.use_gpu:
-            self.error_exit("cannot run Recorder without a GPU")
+        # if not self.use_gpu:
```

Comment:
Add a check whether there is `bug_info[self.key_fix_lines][0]` and I will approve the merge

Reply:
```        
if len(bug_info[self.key_fix_lines]) == 0:
            self.error_exit("no line number to fix")
```

Reply:
Done. Thanks.

Reply:
Thank you for contributing! :)